identity/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "chrome.identity example",
+  "version": "1",
+  "manifest_version": 2,
+  "applications": {
+    "gecko": { "id": "identity.example@mozilla.org" }
```

Comment:
We should set a minimum browser version here of 53? We've also been trying to remove the id, so it can load from web-ext and not block on AMO if you try to sign it.